rides/templates/index.html

```diff
@@ -187,5 +187,7 @@ <h5 class="modal-title">Delete Event</h5>
       {% endfor %}
       {% if current_user.is_authenticated %}<a class="btn btn-primary btn-lg btn-block" align="center" href="/eventform" id="new">Create
       Event</a>{% endif %}
+      <h4 align="center">If you use Rideboard via CSH Login, please ensure your Slack is synced at <a href = "https://eac.csh.rit.edu">https://eac.csh.rit.edu</a>!</h4>
+      <h4 align="center">Re-Login once you have synced to apply changes!</h4>
```

Comment:
Why not check if they're 1. logged in with CSH and 2. don't have slack uid set, and only tell them to visit eac if both are true?

Reply:
https://github.com/ComputerScienceHouse/csh-rideboard/pull/22/commits/94e6d380f1db3a213203cf09c1bae9eae9edb71f